src/lib/native.ts

```diff
@@ -757,3 +757,37 @@ export async function writePref(name: string, value: any) {
         write(file, text.replace(substr, `pref("${name}", ${value})`))
     }
 }
+
+/** Obey Mozilla's orders https://github.com/tridactyl/tridactyl/issues/1800 */
+export async function unfixamo() {
+    try {
+        if (localStorage.unfixedamo === "true") {
+            // unfixamo already ran for the tridactyl instance in this profile
+            return
+        }
+
+        const profile = (await getProfileDir()) + "/"
+        const userjs = await loadPrefs(profile + "user.js")
+        const tridactylPref = "tridactyl.unfixedamo"
+        if (userjs[tridactylPref] === "true") {
+            // unfixamo already ran for this firefox profile
+            return
+        }
+
+        const restricted = "extensions.webextensions.restrictedDomains"
+        const restrictedDomains = '"accounts-static.cdn.mozilla.net,accounts.firefox.com,addons.cdn.mozilla.net,addons.mozilla.org,api.accounts.firefox.com,content.cdn.mozilla.net,discovery.addons.mozilla.org,install.mozilla.org,oauth.accounts.firefox.com,profile.accounts.firefox.com,support.mozilla.org,sync.services.mozilla.com"'
+        if (userjs[restricted] !== undefined && userjs[restricted] !== restrictedDomains) {
```

Comment:
Can't we check if it is === "" since that what we set it to? Some users may have actually added domains to increase security and I don't want to reduce that. (Sorry for lag, got dragged away)